### DISCUSSION TITLE
IP address reveal

### DISCUSSION THREAD

**Discussion by jazzjohn27** [earlephilhower/arduino-pico#1007] [Q&A] [answered]
With the Pico W as a web server with dynamic ip, 
the user needs a way to find the ip without having to log into their router. 

Any ideas?

**Comment by earlephilhower** [comment, marked as answer]
`Serial.println(WiFi.localIP())` or something similar?  You could also hook up a speaker and use https://github.com/earlephilhower/ESP8266SAM to have it say the IP address out loud. :)

**Reply by jazzjohn27** [reply]
Both good ideas. I need to make this as simple as possible for the user, so I'm adding a small display to show the IP address.
[SSD1306 OLED Display Module](url)
The text to speech setup sounds interesting and might be for another project.
Thanks for the response.
